packages/main/src/TabContainer.js

```diff
@@ -226,6 +227,46 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the background color of the Tab Container's header.
+		 *
```

Comment:
On line 195 the header is described as "tab strip":
"Defines the overflow mode of the tab strip"
I suggest to change this sentence (line 195) to:
Defines the overflow mode of the header (the tab strip)